src/main/java/roomescape/reservation/ReservationRepository.java

```diff
@@ -0,0 +1,10 @@
+package roomescape.reservation;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+
+    List<Reservation> findByDateAndTheme_Id(LocalDate date, Long theme);
```

Comment:
파라미터에 null이 들어갈 수 없다면 long으로 표기해주는 것이 명세에 좋습니다. 또한, 파라미터명이 잘못되어 있는 것 같네요!

```suggestion
    List<Reservation> findByDateAndTheme_Id(LocalDate date, long themeId);
```

Reply:
감사합니다 ! 
원시 타입을 쓰면 null이 들어갈 수 없다는 것을 코드로써 확실히 해주니 다른 사람이 볼 때도 좋은 것 같네요 ㅎㅎ

모두 반영했습니다  !